wagon.py

```diff
@@ -746,6 +746,18 @@ def create(source,
     elif 'manylinux' in platform:
         # this is a hack to support Cloudify 5.1, who doesn't handle
         # manylinux wagons. Rename manylinux{1,2010,2014} to linux.
+        # handle new wheel naming convention manylinux*.manylinux*
+        # by getting the last part of platform
+        # example :
+        # >>> platform='manylinux_2_5_x86_64.manylinux1_x86_64'
+        # >>> while 'manylinux' in platform.partition('_')[2]:
+        # ...     platform=platform.partition('_')[2]
+        # >>> platform
+        # '64.manylinux1_x86_64'
+        # >>> platform.partition('_')
+        # ('64.manylinux1', '_', 'x86_64')
+        while 'manylinux' in platform.partition('_')[2]:
+            platform=platform.partition('_')[2]
```

Comment:
```suggestion
            platform = platform.partition('_')[2]
```
huh we dont have flake8 in ci here, and apparently ci doesnt work anyway